addons/web/static/src/js/views/control_panel/control_panel_view.js

```diff
@@ -156,6 +156,9 @@ var ControlPanelView = Factory.extend({
                                 attrs.name ||
                                 attrs.domain ||
                                 'Ω';
+        if (JSON.parse(attrs.modifiers || '{}').invisible) {
+            filter.invisible = true;
```

Comment:
where do you evaluate the condition?

Reply:
@aab-odoo This is done in the model, same behavior as fix of 13.0 https://github.com/odoo/odoo/commit/7bbc5ecca26 

If I have understand correctly the steps are as follow:

- `ControlPanelView.init()`
   - `ControlPanelView._parseSearchArch()`
      - `ControlPanelView._extractAttributes()`
         *here we set `invisible` on filter based on `@modifiers`

   filters then are pushed to `self.loadParams.groups`

- `ControlPanelView.getController()`
   - `ControlPanelView._loadData()`
       which calls `model.load()` with the `loadParams` then calls `model.get()` which will only filter-out invisible filters, see change on the [control_panel_model.js](https://github.com/odoo/odoo/pull/52697/files#diff-ed754dc6a08d6996c2a08477b72394f3R221-R223)

Reply:
@aab-odoo I really want to push this fwport _forward_. Is there any blocking issue ? Do you need more information from me ? Thanks!